### PR TITLE
Clean up: Part 1

### PR DESCRIPTION
**Pull request by Jimbo4350**
Some further removal and condensation of modules: #84

### CODE REVIEW

File: cardano-node/src/Cardano/Node/Configuration/CLI.hs [hunk 1]
```diff
@@ -1,433 +0,0 @@
...
-        <*> lastStrOption
-           ( long "th-period"
-          <> metavar "TH-PERIOD"
-          <> help "Throttle period."
-           )
-        <*> lastIntOption
-           ( long "th-burst"
-          <> metavar "TH-BURST"
-          <> help "Throttle burst."
-           )
```
Comment (mhuesch) [inline cardano-node/src/Cardano/Node/Configuration/CLI.hs]: Where did the contents of this file go? I'm not seeing them relocated anywhere.

Comment (Jimbo4350) [reply cardano-node/src/Cardano/Node/Configuration/CLI.hs]: The exported functions weren't being used anywhere. I need @deepfire to confirm if this is dead code.

Comment (mhuesch) [reply cardano-node/src/Cardano/Node/Configuration/CLI.hs]: Ah ok, nice!

**Pull request merged**